recipes/libssh/all/conanfile.py

```diff
@@ -56,6 +56,8 @@ def requirements(self):
             self.requires("openssl/[>=1.1 <4]")
         elif self.options.crypto_backend == "gcrypt":
             self.requires("libgcrypt/1.8.4")
+        elif self.options.crypto_backend == "mbedtls":
+            self.requires("mbedtls/3.6.0")
```

Comment:
I've tested locally with versions 3.1.0 through 3.6.0 and they all build correctly (but don't pass the test_package step)

Should this requires be a fixed version, or should it be something like `self.requires("mbedtls/[>=3.1.0 <=3.6.0]")`?

Reply:
@obnyis The version range feature is reserved only to few packages yet: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

So, in this specific case, let's use only the latest.